### PR TITLE
Feature/update wallet face

### PR DESCRIPTION
**Pull request by Dragonesis**
- Update UI Wallets components
- Moving Wallets components to the root of components
- Updating components call path
- WalletCard add in storybook
- Add new prop for JFlatButton

### CODE REVIEW

File: src/components/WalletFace/WalletFace.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const WalletFace = ({
               iconName='dots'
               iconSize='medium'
               iconColor='white'
-              hasNotBorder
+              isBordered={false}
```
Comment (IViol) [inline src/components/WalletFace/WalletFace.js]: boolean props should be false by default to avoid `isBordered={false}`
like any other html boolean attrs.

Comment (Dragonesis) [reply src/components/WalletFace/WalletFace.js]: Then I need to specify hasNotBorder or isNotBordered. Else the logic is broken.

File: src/routes/Wallets/routes/Create/components/WalletsCreateView.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// @flow
```
Comment (IViol) [inline src/routes/Wallets/routes/Create/components/WalletsCreateView.js]: this file is not used

File: src/components/base/JIcon/JIcon.js [hunk 3]
```diff
@@ -3,10 +3,8 @@
 import React, { PureComponent } from 'react'
 
 type Props = {
-  // @TODO: need to enumerate all icons
   name: string,
```
Comment (flexxnn) [inline src/components/base/JIcon/JIcon.js]: Please, create JIRA task for this issue (if it is not exists)

**Pull request merged**